assets/models/system/deepseek-r1-distill-llama-8b/model.yaml

```diff
@@ -5,4 +5,4 @@ path:
   type: azureblob
 publish:
   description: description.md
-  type: custom_model
\ No newline at end of file
+  type: custom_model
```

Comment:
This appears to indicate a change even though there isn't one. Could this create issues when publishing?